src/allmydata/immutable/downloader/share.py

```diff
@@ -475,7 +475,9 @@ def _signal_corruption(self, f, start, offset):
         # there was corruption somewhere in the given range
         reason = "corruption in share[%d-%d): %s" % (start, start+offset,
                                                      str(f.value))
-        self._rref.callRemoteOnly("advise_corrupt_share", reason.encode("utf-8"))
+        self._rref.callRemote(
+            "advise_corrupt_share", reason.encode("utf-8")
+        ).addErrback(log.err)
```

Comment:
Passing a descriptive (or at least distinctive) *reason* to `log.err` will make it easier to make sense of any failures logged by this code path.  Can you add that here and to the other new `log.err` errbacks?